manifests/t/TeamRadarr/Radarr/4.5.2.7388/TeamRadarr.Radarr.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 4.5.2.7388

```

```suggestion
PackageVersion: 4.5.2
```
Should match control panel, since previous versions don't use the patch number